.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+<!--
+# Welcome Contributor!
+
+Thank you for contributing to PyEnsign, please follow the instructions below to get
+your PR started off on the right foot.
+
+## First Steps
+
+1. Are you merging from a feature branch into develop?
+
+    _If not, please create a feature branch and change your PR to merge from that branch
+    into the PyEnsign `develop` branch._
+
+2. Does your PR have a title?
+
+    _Please ensure your PR has a short, informative title, e.g. "Enables user to publish to multiple topics at once" or "Corrects bug in Subscriber that causes index error"_
+
+3. Summarize your PR (HINT: See CHECKLIST/TEMPLATE below!)
+-->
+
+This PR fixes #issue_number _(If you are fixing a bug)_ which reported a bug that caused a problem to occur when users...
+
+_(or if you are introducing a new feature)_ which requested a feature to allow the user to...
+
+I have made the following changes:
+
+1.
+2.
+3.
+
+### TODOs and questions
+
+<!--
+If this is a work-in-progress (WIP), list the changes you still need to make and/or questions or the PyEnsign team. You can also mention extensions to your work that might be added as an issue to work on after the PR.
+-->
+
+Still to do:
+
+- [ ]
+- [ ]
+- [ ]
+
+Questions for the @rotationalio/maintainers:
+
+- [ ]
+- [ ]
+
+### CHECKLIST
+
+<!--
+Here's a handy checklist to go through before submitting a PR, note that you can check a checkbox in Markdown by changing `- [ ]` to `- [x]` or you can create the PR and check the box manually.
+-->
+
+- [ ] _Is the commit message formatted correctly?_
+- [ ] _Have you noted the new functionality/bugfix in the release notes of the next release?_
```

Comment:
Do we have a place for release notes? I'm hesitant to include this bit until we do.